config/jobs/kubernetes-sigs/structured-merge-diff/structured-merge-diff-ci.yaml

```diff
@@ -41,3 +41,31 @@ periodics:
   annotations:
     testgrid-dashboards: sig-api-machinery-structured-merge-diff
     testgrid-tab-name: ci-gofmt
+- name: ci-structured-merge-diff-benchmark
+  interval: 12h
+  labels:
+    preset-service-account: "true"
+  decorate: true
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: structured-merge-diff
+    base_ref: master
+    path_alias: sigs.k8s.io/structured-merge-diff
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: golang:latest
```

Comment:
Should this be kubekins-e2e?

Reply:
If this image meets your needs, it's best to use the smaller image.
Consider pinning it to a specific version of go.